src/components/base-page.njk

```diff
@@ -80,7 +80,15 @@
     <script type="text/javascript" src="/public/javascripts/analytics.js"></script>
     <script type="text/javascript" {% if cspNonce %} nonce="{{ cspNonce }}"{%  endif %}>
       window.GOVUKFrontend.initAll()
-      window.DI.appInit({ga4ContainerId: "{{ga4ContainerId}}",uaContainerId:"{{uaContainerId}}"},{disableGa4Tracking:{{ga4Disabled}},disableUaTracking:{{uaDisabled}},cookieDomain:"{{analyticsCookieDomain}}"});
+      window.DI.appInit({
+        ga4ContainerId: "{{ga4ContainerId}}",
+        uaContainerId:"{{uaContainerId}}"
+      },{
+        disableGa4Tracking:{{ga4Disabled}},
+        disableUaTracking:{{uaDisabled}},
+        cookieDomain:"{{analyticsCookieDomain}}",
+        isDataSensitive:"{{analyticsDataSensitive}}"
```

Comment:
@di-aholme  just thought - if the default is set to true. will they need to pass in the parameter, but rather just update to version 2.0.0? so no need to pass a param in unless they want to set it to false?

Reply:
Yeah agreed, this update is only needed for teams that are upgrading GA to `>=2.0.0` but also want to turn off the new `isDataSensitive` flag.